submissions/madmaxWMFU/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,46 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+    name: "Max",
+    gender: "male",
+    species: "human",
+    hands: "2",
+    legs: "2",
+    say: () => `Hallo!`,
+};
+
+const woman = {
+    name: "Lena",
+    gender: "female",
+    species: "human",
+    hands: "2",
+    legs: "2",
+    say: () => `Bonjour!`,
+};
+
+const dog = {
+    name: "Adolf",
+    gender: "male",
+    species: "dog",
+    hands: "0",
+    legs: "4",
+    say: () => `Woof!`,
+};
+
+const cat = {
+    name: "Klara",
+    gender: "female",
+    species: "cat",
+    hands: "0",
+    legs: "4",
+    say: () => `Meeow!`,
+};
+
+// ======== OUTPUT ========
+[man, woman, dog, cat].forEach(arr => print(`${arr.species}; <strong>${arr.name}</strong>; ${arr.gender}; ${arr.hands}; ${arr.legs}; <em>${arr.say()}</em>;`, 'div'));
```

Comment:
We have a pattern string here.
Please implement new function that receive your object and return array joined by `join` method.
In summary your code will be following:
```javascript
[man, woman, etc.].forEach(obj => print(getObjectDescription(obj));
```

Reply:
@o-msh, I created a function which returns an array